client/src/components/ElementEditor/Element.js

```diff
@@ -130,23 +183,59 @@ class Element extends Component {
           editTabs={editTabs}
           previewExpanded={previewExpanded}
           expandable={element.InlineEditable}
+          handleEditTabsClick={this.handleTabClick}
+          activeTab={activeTab}
         />
         <ContentComponent
           id={element.ID}
           fileUrl={element.BlockSchema.fileURL}
           fileTitle={element.BlockSchema.fileTitle}
           content={element.BlockSchema.content}
           previewExpanded={previewExpanded}
+          activeTab={activeTab}
+          onFormInit={() => this.updateFormTab(activeTab)}
         />
       </div>
     );
   }
 }
 
+function mapStateToProps(state, ownProps) {
+  const elementName = loadElementFormStateName(ownProps.element.ID).split('.')[1];
+
+  // InlineEditForm will neither have been rendered nor wrapped in redux-form
+  if (!state.form.formState.element || !state.form.formState.element[elementName]) {
+    return {};
+  }
+
+  const elementId = ownProps.element.ID;
+  const elementFormSchema = loadElementSchemaValue('schemaUrl', elementId);
+
+  const stateValue = state.form.formState.element[elementName].values.Root;
+
+  // Search out a default value for the active tab if it is not already in the state.
+  // {@see Tabs.getDefaultActiveKey}
+  const filterFieldsForTabs = (field) => field.component === 'Tabs';
+  const defaultValue = state.form.formSchemas[elementFormSchema].schema.fields
+    .filter(filterFieldsForTabs)[0].children[0].name;
```

Comment:
You could use `find` instead of `filter` here. Also, it'd probably be good to defensively program in for the possibility there's no tabs - although admittedly there's probably some work to do elsewhere for this...